cloud/types.go

```diff
@@ -236,6 +236,7 @@ type (
 		Reviewers             Reviewers  `json:"reviewers,omitempty"`
 		Status                string     `json:"status"`
 		Draft                 bool       `json:"draft"`
+		ReviewDecision        string     `json:"review_decision"`
```

Comment:
```suggestion
		ReviewDecision        string     `json:"review_decision,omitempty"`
```

Reply:
I changed it as it can be empty, but when I originally added the field I looked at other fields for reference in the same struct that can also be "" I saw description:
```
Description           string     `json:"description"`
```
Is there a difference between the two? I didn't see any, so I when looking at this I'd wonder why one is omitted if empty and the other not.

Reply:
>  Is there a difference between the two? I didn't see any, so I when looking at this I'd wonder why one is omitted if empty and the other not.

Zero-value for description is valid value. Is zero value a valid value for ReviewDecision?

Reply:
It can be "" in case of an error, i.e. rate limit exceeded, but I just tested on a new repository without any review setup and there the API returns null, which will also result in "", so it is a valid value. IMO it's sufficient to treat both cases the same to keep it simple, but if you think it's better to be able to differentiate between the two let me know.

Reply:
I changed it so it will be "" in case of an error and "none" for a the case where there is no review necessary (i.e. the API null case). This way it works similar to other status fields we have. We could use omitempty, but same goes for many other fields in this struct where we don't, so for consistency reasons I did not add it, but we could revise this in a future PR.